banzai/images.py

```diff
@@ -125,15 +136,33 @@ def writeto(self, filename, fpack=False):
                 self.filename += '.fz'
             shutil.move(os.path.join(temp_directory, base_filename), filename)
 
+    def add_data_tables_to_hdu_list(self, hdu_list):
+        """
+        :param hdu_list: a list of hdu objects.
+        :return: a list of hdu objects with a FitsBinTableHDU added
+        """
+        for key in self.data_tables:
+            table_hdu = fits_utils.table_to_fits(self.data_tables[key]._data_table)
+            table_hdu.name = self.data_tables[key].name
+            hdu_list.append(table_hdu)
+        return hdu_list
+
+    def add_bpm_to_hdu_list(self, hdu_list):
```

Comment:
Not sure if / how this will be refactored, but in its current state it should probably be made protected, and come before `add_data_tables`, i.e. be consistent with the order that the methods are called.

Reply:
I will protect both these methods, thanks